Documentation/building/windows-instructions.md

```diff
@@ -26,81 +26,84 @@ the Visual Studio IDE.
 [Building CoreFX on FreeBSD, Linux and OS X](unix-instructions.md)
 ## Tests
 
-We use the OSS testing framework [xunit](http://xunit.github.io/)
+We use the OSS testing framework [xunit](http://xunit.github.io/).
 
 ### Running tests on the command line
 
 By default, the core tests are run as part of the build. Running the tests from
 the command line is as simple as invoking `build.cmd` on windows, and `run-test.sh` on linux and osx. 
 
-You can also run the test for an individual project by building just one test
-project, e.g.:
+You can also run the tests for an individual project by building it individually, e.g.:
 
 ```
 cd src\System.Collections.Immutable\tests
 msbuild /t:BuildAndTest (or /t:Test to just run the tests if the binaries are already built)
 ```
+
 It is possible to pass parameters to the underlying xunit runner via the `XunitOptions` parameter, e.g.:
-````
+```cmd
 msbuild /t:Test "/p:XunitOptions=-class Test.ClassUnderTests -notrait category=outerloop"
-````
+```
 
-In some test directories there may be multiple test projects or directories so you may need to specify the specific test project to get it to build and run the tests.
+There may be multiple projects in some directories so you may need to specify the path to a specific test project to get it to build and run the tests.
 
-Tests participate in the incremental build.  This means that if tests have already been run, and inputs to the incremental build have not changed, rerunning the tests target will not execute the test runner again.  To force re-executing tests in this situation, use msbuild /t:clean;build;test.
+Tests participate in the incremental build.  This means that if tests have already been run, and inputs to the incremental build have not changed, rerunning the tests target will not execute the test runner again.  To force re-executing tests in this situation, use `msbuild /t:clean;build;test`.
 
-The tests can also be filtered based on xunit trait attributes defined in XunitTraitDiscoverers project. These attributes are to be specified over the test method. The available xunit attributes are:
+The tests can also be filtered based on xunit trait attributes defined in [`xunit.netcore.extensions`](https://github.com/dotnet/buildtools/tree/master/src/xunit.netcore.extensions). These attributes are to be specified over the test method. The available attributes are:
 
-_**OuterLoop:**_
-This attribute returns the 'outerloop' category, so to run outerloop tests use the following commandline,
-```
+_**`OuterLoop`:**_
+This attribute applies the 'outerloop' category; to run outerloop tests, use the following commandline
+```sh
```

Comment:
You're right; from context this is on Windows.  Below I used `sh` where commands seemed likely to be run on a Linux machine.  For these command lines I'm not sure there's any highlight difference at all between the two.